addons/phone_validation/lib/phonenumbers_patch/__init__.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2009 The Libphonenumber Authors
```

Comment:
Just having a question on this license stuff ... I don't know anything about licenses. Is it normal to have that here ? Is it really copyrighted by "libphonumber authors" ?

Reply:
There is some lines copied verbatim from https://github.com/daviddrysdale/python-phonenumbers/blob/v8.12.32/python/phonenumbers/data/__init__.py that have this license notice in the file.

But the license doesn't really concerns any line in the file but the original data (https://github.com/google/libphonenumber/blob/master/resources/PhoneNumberMetadata.xml) that is used to auto-generate the `region_*.py` files.

I did not think that was necessary, but since `python-phonenumbers` is doing it, we might as well do it too.

Reply:
It's about this part of the license: https://github.com/daviddrysdale/python-phonenumbers/blob/v8.12.32/LICENSE#L94-L95
Yeah, we just copied the auto-generate part of the code, but since the generator is under this license, it seems logical to me to respect this license.

Reply:
Ok, I was just highlighting it as I don't know anything about licenses. Everything seems ok then, don't know who should r+ but I think we can send :) .